ray-operator/controllers/ray/common/pod.go

```diff
@@ -137,6 +148,15 @@ func configureGCSFaultTolerance(podTemplate *corev1.PodTemplateSpec, instance ra
 					})
 				}
 			} else {
+				// If users directly set the `redis-username` in `rayStartParams` instead of referring
```

Comment:
I am considering not adding any specific logic for `redis-username` (i.e. remove the change from L151 to L159) in the old configurations to avoid maintenance costs and to provide an incentive for users to migrate to the new API, `GcsFaultToleranceOptions`. HDYT @rueian?

Reply:
ok, then should we make the incentive clear in the `validateRayClusterSpec`?